Bottom: 5b03f3f48513c523fc196edd34da879c2adf6658
Top:    aaab480366f995abe95b3f80e6641f6c642a896b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 13:58:38 +0100

Refresh of add-strip-hierarchy-options-to

---

diff --git a/src/action.c b/src/action.c
index e152d59..eec75cb 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4717,7 +4717,7 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
   char *function = ARG (0);
   if (function && PCB->ElementOn)
     {
-      HideCrosshair (True);
+      HideCrosshair (true);
       switch (GetFunctionID (function))
 	{
 	case F_Object:
@@ -4742,7 +4742,7 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 	case F_SelectedElements:
 	case F_Selected:
 	  {
-	    Boolean changed = False;
+	    bool changed = false;
 	    ELEMENT_LOOP (PCB->Data);
 	    {
 	      if ((TEST_FLAG (SELECTEDFLAG, element) ||
@@ -4756,7 +4756,7 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 		  TOGGLE_FLAG (STRIPHIERFLAG, element);
 /* TODO: 		   SetTextBoundingBox (&PCB->Font, new); */
 		  DrawElementName (element, 0);
-		  changed = True;
+		  changed = true;
 		}
 	    }
 	    END_LOOP;
@@ -4767,7 +4767,7 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 	      }
 	  }
 	}
-      RestoreCrosshair (True);
+      RestoreCrosshair (true);
     }
   return 0;
 }
